pkgs/tools/inputmethods/keyd/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, python39Packages, python39, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "keyd";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "rvaiya";
+    repo = "keyd";
+    rev = "v${version}";
+    sha256 = "sha256-YZSwXDrzly5OlNh8j+Y2/qYmGHXE5T3ZAvq7e45jVL0=";
+  };
+
+  makeFlags = [ "DESTDIR=${placeholder "out"}" ];
```

Blanking PREFIX is actually necessary, since the Makefile [sets it](https://github.com/rvaiya/keyd/blob/master/Makefile#L3). Suggest reverting that removal.

```suggestion
  makeFlags = [ "DESTDIR=${placeholder "out"}" "PREFIX=" ];
```